pkg/agent/runner.go

```diff
@@ -807,7 +813,8 @@ func (r *Runner) DoSchedulerRequest(
 		return nil, fmt.Errorf("Bad JSON response: %w", err)
 	}
 
-	logger.Info("Received response from scheduler", zap.Any("response", respData))
+	// there will be "Plugin request successful" INFO log right after
```

Comment:
I found this comment a little confusing ("where?" ; "in what cases?"). IMO it's ok to say nothing here, but adding more context about where the log line comes from would also be ok